modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -11,11 +11,15 @@ module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::Veterans::Base
         skip_before_action :validate_json_format
         before_action :set_lighthouse_claim, :set_bgs_claim!, :verify_if_dependent_claim!
+        FORM_NUMBER = '5103'
 
         def submit
           validate_veteran_name(false)
 
+          tracked_item_ids = params['data']['attributes']['trackedItemIds'] if params['data'].present?
```

Comment:
@stiehlrod  I think we need to make sure ['attributes'] is in place here.  Right now this errors out if I send in 
```
{
    "data": {
        "type":"form/5103"
    }
}
```
Which is allowed, maybe as simple as just updating the conditional to `if params[data][attributes].present` since that is what we are really checking anyways ?